### DISCUSSION TITLE
Submenu

### DISCUSSION THREAD

**Discussion by alekseykurylev** [adobe/react-spectrum#5607] [Q&A] [answered]
![image](https://github.com/adobe/react-spectrum/assets/8526127/f14b3106-2e0c-40c6-b640-9790b0fd441e)

Can you tell me how to implement submenu using react-aria-components correctly?

**Comment by LFDanLu** [comment, marked as answer]
Work is currently underway to support submenus in React Aria Components: https://github.com/orgs/adobe/projects/19/views/4?pane=issue&itemId=43944585, so stay tuned!

**Reply by alekseykurylev** [reply]
thanks
